email_mailbox/src/components/SettingGeneralWrapper.js

```diff
@@ -272,6 +297,13 @@ class SettingGeneralWrapper extends Component {
       newTwoFactorParams.isLoading = false;
       setTwoFactorAuthStatus(nextProps.twoFactorAuth);
     }
+    if (
+      nextProps.readReceiptsEnabled &&
+      this.state.readReceipts.enabled !== nextProps.readReceiptsEnabled
+    ) {
+      newReadReceipts.enabled = nextProps.readReceiptsEnabled;
+      setReadReceiptsStatus(nextProps.readReceiptsEnabled);
```

Comment:
Where does readReceiptsEnabled come from? I don't think you should be updating localStorage if this is not the root of the value change. Shouldn't you just receive the prop to display? I've not enough context so i might be wrong

Reply:
[Refactoring Issue](https://github.com/Criptext/Criptext-Email-React-Client/issues/542)